synapse/__init__.py

```diff
@@ -21,6 +21,7 @@
 # the built-in cortex modules...
 BASE_MODULES = (
     ('synapse.models.syn.SynMod', {}),
+    ('synapse.models.axon.AxonMod', {}),
```

Comment:
It is a data model so its appropriate to track there (as we could have model revisions over time) which need migrations done.  Doing it via the source makes that less flexible.